src/Symfony/Component/Yaml/Parser.php

```diff
@@ -167,12 +167,17 @@ private function doParse($value, $flags)
         $this->currentLine = '';
         $value = $this->cleanup($value);
         $this->lines = explode("\n", $value);
-        $this->locallySkippedLineNumbers = [];
 
+        $this->locallySkippedLineNumbers = [];
         if (null === $this->totalNumberOfLines) {
             $this->totalNumberOfLines = \count($this->lines);
         }
 
+        // This a quoted string that spans multiple lines
+        if (false !== $pos = strpos($value, "'\n")) {
+            throw new ParseException(sprintf('A quoted string that spans multiple cannot be parsed at position %s', $pos), -1, $value);
```

Comment:
`A quoted string that spans multiple lines is not supported ...`?  so it's clear for the user the parser is at fault